browser/components/molecules/PendingPrrrs/index.js

```diff
@@ -19,6 +19,7 @@ export default class PendingPrrrs extends Component {
 
   claimPrrr(prrr){
     claimPrrr(prrr.id)
+    //what is the underscore in the line below doing?
```

Comment:
Yes I found out it's a newish was to just declare a placeholder empty parameter. Took out this commit in later commits 😎